src/com/geekhub/hw4/set/SetOperationsUtility.java

```diff
@@ -0,0 +1,53 @@
+package com.geekhub.hw4.set;
+
+import java.util.Set;
+
+public class SetOperationsUtility implements SetOperations {
+
+    @Override
+    public boolean equals(Set a, Set b) {
+        return a.equals(b);
+    }
+
+    @Override
+    public Set union(Set a, Set b) throws IllegalAccessException, InstantiationException {
```

Comment:
I tried to identify class of "a" and used newInstance(), but after reading some docs understood my mistake and fixed all methods like this:
        Set set = new HashSet<>();
        set.addAll(a);
        set.addAll(b);
        return set;
Is it right?